lib/src/tasks/task_runner/api.dart

```diff
@@ -77,42 +71,29 @@ class TaskGroup {
   }
 
   /// Begin each subtask and wait for completion
-  Future start() async {
+  Future run() async {
     List<Future> futures = <Future>[];
     var timer = new Timer.periodic(new Duration(seconds: 30), (_) {
       reporter.log('Tasks running...');
     });
     for (SubTask task in subTasks) {
       task.startProcess();
-      futures.add(task.taskProcess.exitCode);
+      task.taskProcess.exitCode.then((int exitCode) {
+        reporter.log(task.taskOutput);
+        if (exitCode != 0) {
+          successful = false;
+          reporter.log(task.taskError);
+          this.taskGroupStderr +=
+              'The command, ${task.command}, contained this in the stderr; ${task.taskOutput}\n ${task.taskError}\n';
```

Comment:
`task.taskOutput` is logged above on line 82, so wouldn't this be logging that same info twice?